sql/expression/function/logarithm.go

```diff
@@ -0,0 +1,223 @@
+package function
+
+import (
```

Comment:
About the style in the import block, could you separate the standart libraries from the third libraries by a blank line as below?
```go
import (
        "std1"
        "std2"

        "third1"
        "third2"
)
```